### PR TITLE
Update collection properties

### PR DESCRIPTION
**Pull request by CaroDev17**
Update collection models and propertis..!

### CODE REVIEW

File: models/UsersModel.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose');
+
+var modelsRegistro = mongoose.Schema({
```
Comment (pixrael) [inline models/UsersModel.js]: please change the name of "modelsRegistro" to "registerModel"

File: models/UsersModel.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    id: Number,
+    name: String,
+    lastName: String,
+    mail: String,
```
Comment (pixrael) [inline models/UsersModel.js]: please change the name of this property from "mail" to "email"

File: routes/index.js [hunk 3]
```diff
@@ -24,8 +24,8 @@ router.post("/api/nuevo", function (req, res, next) {
   console.log("--> ", req.body);
```
Comment (pixrael) [inline routes/index.js]: Please, remove this and all the console logs in the code, but those that tells status of the db conection and port where is runnig the serve

**Review by pixrael — CHANGES_REQUESTED** [review]
Great the Pull request, only pending little changes

File: models/UsersModel.js (hunk 1 above)
Comment (CaroDev17) [reply models/UsersModel.js]: Done!

File: models/UsersModel.js (hunk 2 above)
Comment (CaroDev17) [reply models/UsersModel.js]: Done!

File: routes/index.js (hunk 3 above)
Comment (CaroDev17) [reply routes/index.js]: Done!

**Review by pixrael — APPROVED** [review]

**Pull request merged**